### ISSUE TITLE
Method does not always return null

### ISSUE THREAD

**Issue by shanethehat** [scrutinizer-ci/php-analyzer#758]
php analyser has a flagged a line similar to `$value = $this->getValue();` as always returning null, but this is not the case. The method in question looks something like:

```
public function getValue($argument) {
    if ($argument == 'something') {
        return null;
    }

    try {
        return new Instance();
    } catch (Exception $e) {
        return null;
    }
}
```

It appears that the return inside the try block is not detected. 

see https://scrutinizer-ci.com/g/inviqa/graze/inspections/734dbe40-4392-487d-adbd-c381b29314d8/issues/files/app/code/Graze/ProductInstagramImages/Model/DirectLinkConverter.php?status=new&orderField=path&order=asc&issueId=dfd9c976-2986-483c-9688-1ade83269000
